crates/matrix-sdk/src/client/mod.rs

```diff
@@ -101,8 +108,8 @@ const DEFAULT_UPLOAD_SPEED: u64 = 125_000;
 const MIN_UPLOAD_REQUEST_TIMEOUT: Duration = Duration::from_secs(60 * 5);
 
 type EventHandlerFut = Pin<Box<dyn Future<Output = ()> + Send>>;
-type EventHandlerFn = Box<dyn Fn(EventHandlerData<'_>) -> EventHandlerFut + Send + Sync>;
-type EventHandlerMap = BTreeMap<(EventKind, &'static str), Vec<EventHandlerFn>>;
+pub(crate) type EventHandlerFn = dyn Fn(EventHandlerData<'_>) -> EventHandlerFut + Send + Sync;
```

Comment:
Is the addition of `pub(crate)` still needed with the current revision of this PR?

Reply:
Yes, because the type is used in `Client.register_event_handler` to annotate the content of the Box which is passed to `EventHandlerWrapper`.

Reply:
But `register_event_handler` is defined in the same module.

Reply:
Right, sorry, I mixed it up. It is used to define the type of `EventHandlerWrapper.handler_fn` in the event_handler module.